Makefile

```diff
@@ -30,15 +30,15 @@ PYTHON_TARGET := 2.7
 REQUIREMENTS := test-requirements.txt requirements.txt
 PIP_OPTIONS := $(ST2_PIP_OPTIONS)
 
-NOSE_OPTS := --rednose --immediate
+NOSE_OPTS := --rednose --immediate --with-parallel
 NOSE_TIME := $(NOSE_TIME)
 
 ifdef NOSE_TIME
-	NOSE_OPTS := --rednose --immediate --with-timer
+	NOSE_OPTS := --rednose --immediate --with-parallel --with-timer
 endif
 
 ifndef PIP_OPTIONS
-	PIP_OPTIONS := -U -q
```

Comment:
`removed --upgrade` flag which really slow & eats big chunk of timing.
This especially makes sense if we follow strict (`==`) pip fixation in `fixed-requirements`.